lib/active_admin/views/index_as_table.rb

```diff
@@ -256,9 +256,9 @@ def selectable_column(**options)
         end
 
         # Display a column for the id
-        def id_column
+        def id_column(title: resource_class.human_attribute_name(resource_class.primary_key))
```

Comment:
I'm wondering if this should have a similar signature to the `column` method for consistency where the first argument is the title (no kwarg) if given, otherwise if not then it's treated as the options hash (keyword arguments) instead.

https://github.com/activeadmin/activeadmin/blob/bb1bade56d7da250f1fb3ae66a69a885a42e98e9/lib/active_admin/views/components/table_for.rb#L34-L37

Reply:
You are right. It makes sense to match the column signature. Users likely try that signature.